### ISSUE TITLE
QRcode binary content

### ISSUE THREAD

**Issue by laruche** [EddyVerbruggen/nativescript-barcodescanner#186]
Hi 

Congrats for your project :) 
I'm just make a try with a special implementation of QRcode we use in entreprise solution base on QRcode documentation 

We are using qrcode with binary content
YOu can read it with scandit app or ZXing 
https://zxing.org/w/decode.jspx

There is a way to implemante this code in your plugin ? 
Only test in IOS (your are using built in qr decoder ?)
![demo](https://user-images.githubusercontent.com/1287795/52574202-b5270380-2e13-11e9-9603-4f9df64a44c0.png)

**Comment by EddyVerbruggen** [comment]
Yes, the built-in decoder is used on iOS. On Android, a fork of ZXing is used. I guess the Android code can be updated to support it, but on iOS we need to work with whatever iOS gives us.

Alternatively, you may have more luck scanning this code with https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/master/docs/ML_KIT.md#barcode-scanning

**Comment by laruche** [comment]
I will try ML_KIT 
but at this time compilation failing on my mac :/ 

possible to use Zxing wich support this kind of code on IOS ?
